src/utils/eventMapper.js

```diff
@@ -0,0 +1,16 @@
+const EVENTS_LIST = ['change', 'keyup', 'keypress', 'keydown', 'focus', 'focusin', 'focusout', 'blur'];
+
+function mapEvents(config) {
```

Comment:
Konsekwencje tego co pisałem w oInput.js . Jak wywołujesz tą metodę z config=undefined to Object.entries wywali błąd.

Reply:
Racja, poprawię to.